src/playback-watcher.js

```diff
@@ -220,14 +219,6 @@ export default class PlaybackWatcher {
     this.tech_.trigger({type: 'usage', name: `vhs-${type}-download-exclusion`});
 
     if (type === 'subtitle') {
```

Comment:
Might be worth having a comment here. Something like `// subtitles aren't guaranteed to be present for every bit of video, therefore, we can't determine whether downloads have stalled based on the subtitle's buffer`